modules/smaatoBidAdapter.js

```diff
@@ -141,7 +141,8 @@ export const spec = {
           meta: {
             advertiserDomains: bid.adomain,
             networkName: bid.bidderName,
-            agencyId: seatbid.seat
+            agencyId: seatbid.seat,
```

Comment:
pls update 
`const SMAATO_CLIENT = 'prebid_js_$prebid.version$_3.0'` to version 3.1 now

Reply:
updated